apis/low_level_debug/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#![no_std]
+
+use libtock_platform::{CommandReturn, Syscalls};
+
+/// The low-level debug API provides tools to diagnose userspace issues that make normal debugging workflows (e.g. printing to the console) difficult.
```

Comment:
IIRC it's standard Rust style to wrap comments at 80 characters (I'm surprised `rustfmt` doesn't wrap them by default). Please line-wrap your comment blocks.

Reply:
Yes, though it's 80 characters long not including preceding whitespace, not exceeding the 100 char line limit. Auto-wrapping can be enabled with the [`wrap_comments`](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#wrap_comments) flag. Should that be added to the root `rustfmt.toml`?

Reply:
> Yes, though it's 80 characters long not including preceding whitespace

Oh. That's definitely not what I've been doing, I've been wrapping at 80 columns including preceding whitespace. I don't even know if my editor can do that.

Maybe? Definitely doesn't fit in this PR, as it would likely end up changing a lot of code.

Reply:
Regardless, this line is 150 columns wide, and I think it be wrapped at 100 columns at the widest.

Reply:
Erm oops, I missed the latest commit where you fixed this :-(. Sorry for asking again for a change you already made.